dotcom-rendering/src/components/CrosswordInstructions.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+import {
+	textEgyptian17,
+	textEgyptianBold17,
+} from '@guardian/source/foundations';
+
+const instructionsStyles = css`
+	${textEgyptian17};
+	white-space: pre-line;
+`;
+
+const headerStyles = css`
+	${textEgyptianBold17};
+	white-space: pre-line;
+`;
+
+export const CrosswordInstructions = ({
+	instructions,
+	className,
+}: {
+	instructions: string;
+	className?: string;
+}) => (
+	<div css={instructionsStyles} className={className}>
```

Comment:
Can we discuss the "CSS override" pattern, also with @SiAdcock , as it's something that we've recommended against elsewhere[^1]?

[^1]: https://github.com/guardian/dotcom-rendering/pull/12496#discussion_r1838781643

Reply:
In this case @JamieB-gu are you referring to the `className` being passed as a prop?

Reply:
i'll leave this out for now. we need to do a design pass on the whole page layout, so this might not be needed then anyway